### ISSUE TITLE
Error in tnrs_match_names() for unmatched names

### ISSUE THREAD

**Issue by FlukeAndFeather** [ropensci/rotl#132]
Unhelpful (and likely unintended) error in tnrs_match_names() when there's no match. Reprex below.

``` r
rotl::tnrs_match_names("thisissomenonsense")
#> Warning in check_tnrs(res): thisissomenonsense are not matched
#> Error in names(summary_match) <- c(names(tnrs_columns), "number_matches"): 'names' attribute [7] must be the same length as the vector [0]
```

<sup>Created on 2020-10-09 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by FlukeAndFeather** [comment]
Based on [test-match_names.R](https://github.com/ropensci/rotl/blob/830ae0508bbdec9d52d806a89edb0eb0f6f77fc3/tests/testthat/test-match_names.R#L105-L115) it looks like this is intended behavior. If that's the case, I suggest a more informative error message. But I don't think this should be the intended behavior because if there's one invalid taxon in a list of 1,000 taxa then I want a result with 999 matches. Otherwise I have to iteratively prune the invalid taxa until I get a good result.

**Comment by FlukeAndFeather** [comment]
Ah, this seems to be fixed by 4308b5f. I'm not seeing this error if I use the dev version of rotl. Any idea when this will be on CRAN?

**Comment by fmichonneau** [comment]
The latest version of rotl was just accepted on CRAN. Wait a few days for this new version to be available as binary and in a CRAN mirror near you. Thanks!
